meta-oe/recipes-connectivity/hiba/hiba_git.bb

```diff
@@ -0,0 +1,40 @@
+
+SUMMARY = "Google Host Identity Based Authorization"
+DESCRIPTION = "HIBA is a system built on top of regular OpenSSH \
+  certificate-based authentication that allows to manage flexible \
+  authorization of principals on pools of target hosts without the need to \
+  push customized authorized_users files periodically."
+
+S = "${WORKDIR}/git"
+
+LICENSE = "BSD"
```

Comment:
Be specific about which BSD license e.g `BSD-3-Clause`

Reply:
Done.